pages/tools/sdk.md

```diff
@@ -36,6 +37,9 @@ If you feel something is missing, not clear or could be improved, please don't h
             * [.move(uuid, application)](#resin.models.device.move) ⇒ <code>Promise</code>
             * [.restart(uuid)](#resin.models.device.restart) ⇒ <code>Promise</code>
             * [.reboot(uuid)](#resin.models.device.reboot) ⇒ <code>Promise</code>
+            * [.shutdown(uuid)](#resin.models.device.shutdown) ⇒ <code>Promise</code>
```

Comment:
They are pulled.

On Sat, Apr 23, 2016 at 8:56 PM Shaun Mulligan <notifications@github.com>
wrote:

> In pages/tools/sdk.md
> <https://github.com/resin-io/docs/pull/202#discussion_r60832354>:
>
> > @@ -36,6 +37,9 @@ If you feel something is missing, not clear or could be improved, please don't h
> >              * [.move(uuid, application)](#resin.models.device.move) ⇒ <code>Promise</code>
> >              * [.restart(uuid)](#resin.models.device.restart) ⇒ <code>Promise</code>
> >              * [.reboot(uuid)](#resin.models.device.reboot) ⇒ <code>Promise</code>
> > +            * [.shutdown(uuid)](#resin.models.device.shutdown) ⇒ <code>Promise</code>
>
> Were these changes made directly on sdk.md? i think these are pulled in
> dynamically no?
>
> —
> You are receiving this because you were assigned.
> Reply to this email directly or view it on GitHub
> <https://github.com/resin-io/docs/pull/202/files/6ce650f0d6ab30bee7d6e681703d942eb19b2519#r60832354>
>